docs/modules/integrations/pages/zephyr-exporter.adoc

```diff
@@ -0,0 +1,160 @@
+= Zephyr Exporter
+
+Zephyr Exporter is a tool used for exporting test execution results into Jira Zephyr Plugin.
+
+Features:
+
+* Create test executions
+* Set test execution statuses
+
+== Zephyr Export Properties
+
+[cols="1,2,3", options="header"]
+|===
+
+|Property
+|Required
+|Description
+
+|`zephyr.exporter.source-directory`
+|true
+|Path to directory with test execution JSON results.
+
+|`zephyr.project-key`
+|true
+|Key of a Jira project where new test executions should be created
+
+|`zephyr.version-name`
+|true
+|Version/release name where new test executions should be created
+
+|`zephyr.cycle-name`
+|true
+|Cycle/sprint name where new test executions should be created
+
+|`zephyr.folder-name`
+|true
+|Folder name where new test executions should be created
+
+|===
+
+== Zephyr Execution Status Mapping
+
+The Zephyr plugin for Jira has own configurable execution statuses. For getting detailed information about them can be used request like https://jira.example.com/rest/zapi/latest/util/testExecutionStatus. These properties are used to establish a correspondence between Vividus and Zephyr execution statuses.
```

Comment:
`These properties are used to establish a correspondence between Vividus and Zephyr execution statuses.` ->
`The following properties are used to setup a mapping between Vividus and Zephyr execution statuses.`